packages/swingset-theme-hashicorp/src/components/side-nav/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { NavigationTree } from 'swingset/types'
+import Category from './category'
+
+type SideNavBarProps = {
+  categories: NavigationTree
+}
+
+function SideNavigation(props: SideNavBarProps) {
+  const { categories } = props
```

Comment:
nit, we can destructure this:
```suggestion
function SideNavigation({ categories }: SideNavBarProps) {
```